docs/processes/release/tasks/release-build.md

```diff
@@ -24,185 +24,40 @@ License & terms of use: http://www.unicode.org/copyright.html
 
 ## Building ICU4J Release Files
 
-<span style="color:red">*Note: This instruction is reviewed for ICU4J 69.1. Some
-description may not apply to previous ICU releases. ICU4J library (as of ICU 69)
-supports JRE 7 or later Java runtime environment. To avoid the Java's system
-library class's backward compatibility problem, ICU4J binary files must be
-compiled with Java system libraries from JRE 7.*</span>
+<span style="color:red">*Note: This instruction is reviewed for ICU4J 78.1. Some
+description may not apply to previous ICU releases. If you need instructions on
+how to release older versions see the history of this file.*</span>
```

Comment:
I don't think that Markdown symbols are honored inside HTML tags. So remove the pair of asterisks and replace with the appropriate `i` or `b` tag or update the `span` style attribute, if desired.

Reply:
Done